parlai/scripts/build_pytorch_data.py

```diff
@@ -26,8 +26,10 @@
 
 
 def get_pyt_dict_file(opt):
-    if os.path.exists(opt.get('dict_file', '')):
+    if opt.get('dict_file') and os.path.exists(opt.get('dict_file')):
         return opt['dict_file']
+    if opt.get('dict_file', None) is None and opt.get('model_file'):
```

Comment:
maybe just `opt.get('dict_file') is None and opt.get('model_file')`; the None is the default argument for `opt.get`

Reply:
right, i'll change that